doc/sphinx/source/project/introduction.rst

```diff
@@ -4,176 +4,86 @@
-manually integrated with each hardware-specific kernel library. Each integration 
+.. figure:: ../graphics/intro_kernel_to_fw_accent.png
+   :width: 555px
+   :alt: 
+      
+Each individual framework must be manually integrated with each hardware-specific kernel library. Each integration 
 is unique to the framework and its set of deep learning operators, its view on 
 memory layout, its feature set, etc. Each of these connections, then, represents 
 significant work for what will ultimately be a brittle setup that is enormously 
-expensive to maintain.    
+expensive to maintain.  
 
-.. _figure-F:
+nGraph solves this problem with nGraph bridges that connect to the deep learning frameworks. nGraph bridges take computational graphs from supported deep learning frameworks, and they reconstruct the graph using nGraph IR with a few primitive nGraph operations. With the unified computational graph, kernel libraries no longer need to be separately integrated to each deep learning frameworks. Instead, the libraries only need to support nGraph primitive operations, and this approach streamlines integration process for the backend.  
 
-.. figure:: ../graphics/dl-current-state.png
-   :width: 700px
-   :alt: Scalability matters
-
-   Figure F: The number of kernels necessary to achieve optimal performance is 
-   bounded by the product of the number of chip designs one wishes to support, 
-   the number of data types supported, the number of operations, and the 
-   cardinality of each parameter for each operation.
-
-In the past, this upper bound was quite limited; however, since the industry is 
-shifting toward a more diverse future in terms of deep learning hardware, the 
-number of distinct kernels is exploding and will continue to explode.
-
-
-Get the best of both worlds
----------------------------
+3. Increasing number of kernels 
+---------------------------------------------------------
 
-Integrating a framework on nGraph can be an attractive option for hardware 
-companies trying to design their own deep learning hardware or network architecture. 
-Framework integration is non-trivial amount of work, and nGraph automatically 
-does much of the heavy lifting. Furthermore, PlaidML can provide a wide range of 
-hardware coverage and optimization automatically. Any hardware that supports 
-LLVM, OpenCL, OpenGL, CUDA or Metal can be supported automatically with PlaidML 
-and nGraph.  
+As mentioned in the pervious section, kernel libraries need to be integrated with multiple deep learning frameworks, and this arduous task becomes even harder due to increased numbers of required kernels for achieving optimial performance. The number of required kernels is product of number of chip designs, data types, operations, and the cardinality of each parameter for each operation. In the past, the number of required kernels was limited, but as the AI research and industry rapidly develops, the final product of required kernels is increasing exponentially. 
```

Comment:
Delete "As mentioned in the pervious section".  

To transition in to this last part, maybe say something like "All of this adds up to arduous work as kernel libraries need to be integrated..."